docs/diagrams/orchestration/identity/identity.mmd

```diff
@@ -33,6 +35,7 @@ sequenceDiagram
     spot -) orch_api : Return a signed Core Identity JWT
     orch_api ->> orch_api : Store signed Core Identity JWT
     auth ->> orch_api : Poll for updates
+    orch_api -->> ais : Check for interventions
```

Comment:
Should this come after the COMPLETED response?

Reply:
It looks like Orch check for interventions before returning the COMPLETED response back to Auth.

The response is here https://github.com/govuk-one-login/authentication-api/blob/7f26ad1d332654ec038ad26ba51591a3c868c6fa/ipv-api/src/main/java/uk/gov/di/authentication/ipv/lambda/ProcessingIdentityHandler.java#L194 - AIS is checked on L181 above